.github/workflows/build.yml

```diff
@@ -63,13 +63,13 @@ jobs:
             # Determine the repo and leave that unset to use the normal checkout behavior
             # of using the merge commit instead of HEAD
             case $GITHUB_REPOSITORY in
-              "apache/incubator-nuttx")
+              "tiiuae/nuttx")
```

Comment:
why change this?

Reply:
Looks like this commit was not meant to be for the upstream repo.

Reply:
Oops, I am terribly sorry. I fumbled with repositories, this was indeed not intended for upstream repo!